api/content/data/content.rng

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+         xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
+
+    <include href="../../../core/data/core.rng"/>
+
+    <start>
+        <ref name="libero.content"/>
+    </start>
+
+    <div>
+
+        <a:documentation>content element</a:documentation>
+
+        <define name="libero.content">
+
+            <element name="content">
+
+                <oneOrMore>
+                    <ref name="libero.anything.element"/>
```

Comment:
Seems hard to try and include the other schemas inside, so this will rely on the implementer breaking it up and validating the pieces individually (and checking for duplicates etc).

Reply:
Haven't seen a sample but how do the sub elements indicate their schema? If the first element of `content` is something like a `<table>` I would expect a validation library to in any case to fail on `<table><p></p></table>` even if this containers says this is a list of `anything`.

Reply:
Can look at https://github.com/libero/walking-skeleton/blob/master/schemas/data/core/inline.rng for examples, they declare what they can contain like anything else.

Reply:
Translating to myself and @jhroot which is about to implement this:

- implementer should break up the `<content>` into parts
- implementer should store the parts separately so that they can be served one at a time
- each part may be validated against the schema because the element name defines what it can contain, so `<front>` can contain only some stuff and not just any generic XML (out of scope for the first API PR)
- there should be no duplicate parts, like two `<front>` elements (out of scope for the first API PR)

Reply:
(actually there can be multiple `<front>` with different languages: all the more reason to reserve validations to their own follow-up PRs)